tools/import-owner-private-keys.sh

```diff
@@ -1,6 +1,100 @@
 #!/bin/bash
+# This script will generate a Key-Pair for Owner Attestation.
+
+if [[ "$1" == "-h" || "$1" == "--help" ]]; then
+    cat << EndOfMessage
+Usage: ${0##*/} [<owner-keys-tar-file>]
+
+Arguments:
+  <owner-keys-tar-file>  A tar file containing the 3 private keys and associated 3 certs
+
+Required environment variables:
+  SDO_KEY_PWD - The password for your generated keystore. This password must be passed into run-sdo-owner-services.sh if you are passing in your own master keystore. This will allow it to be mounted to $containerHome/ocs/config/application.properties/fs.owner.keystore-password
```

Comment:
SDO_KEY_PWD is passed in as an environment variable for this script. In `run-sdo-owner-services.sh` if you pass in your own master keystore then you must also pass in the corresponding SDO_KEY_PWD env variable. Once that is passed in it will update ocs/config/application.properties/fs.owner.keystore-password with whatever password you decided to use. This is the same password that will be used by this script so that it can insert the keypairs inside the master keystore.

In short you should be able to use our master keystore or even pass in your own with a custom password and everything should continue to work